### ISSUE TITLE
Add Bot/User Avatar Initials Text and Background Color to defaultStyleOptions

### ISSUE THREAD

**Issue by mutanttech** [microsoft/BotFramework-WebChat#1904]
I want to customize the background and text color of Bot/User Avatar Initials based on the overall color scheme of the webchat ui. 
I only see customizing the css as a solution but getting it from defaultStyleOptions would be way to benefit everyone.

**Comment by corinagum** [comment]
Thanks for filing this issue. For clarification and per your discussion with [RonHouben](https://github.com/Microsoft/BotFramework-WebChat/issues/1831#issuecomment-482958541), do you wish to be able to style this _apart_ from the accent color?

**Comment by mutanttech** [comment]
@corinagum, Yes, the styling from accent color is not really fit for purpose. And while we are providing granular control to specify avatar text and image, the styling control will actually improve the usage.

**Comment by compulim** [comment]
@reyesrico after talking offline, this is the stuff you will need.

Proposed list of style options:

```js
  botAvatarBackgroundColor: null,
  botAvatarBorderRadius: '50%',
  botAvatarFont: null,
  botAvatarFontSize: null,
  botAvatarTextColor: 'White',
```

(Also same to `userAvatar*`)

Please test the following cases:

- The developer did not modify any styles mentioned above
- The developer modified all of the style above, with different settings for bot side and user side
- The developer modified `accent` but not `botAvatarBackgroundColor`, we should use `accent` for the avatar color instead
   - For the same reason, we are not doing a great job at suggested actions, please file a bug
   - Also same for `primaryFont` vs. `botAvatarFont`

To build incrementally:

- `npm run build` (only needed for the first time after you clone the branch)
- `npm run watch`

For testing, the easiest way (to me) is to hack our `04.a` sample:

- Run `npx serve`
- Modify `/samples/04.a.display-user-bot-initials-styling/index.html`
   - Change `src` of `<script>` from `cdn.botframework.com` to `/webchat.js`
- Browse to http://localhost:5000/samples/04.a.display-user-bot-initials-styling/index.html

To run tests:

- Install Docker
- Run `docker-compose up --build`, and restart it every time you made a change to Web Chat
- Run `npm test`
- All existing tests should pass

**Comment by tdurnford** [comment]
PR https://github.com/microsoft/BotFramework-WebChat/pull/2384 added `botAvatarBackgroundColor` and `userAvatarBackgroundColor`. Do we still want to add `botAvatarFont` and `userAvatarFont` to the default style options?

**Comment by corinagum** [comment]
Hmm it doesn't seem like we've had much demand for these props yet. But if @reyesrico is willing to help, maybe we should do it? @compulim, thoughts?

**Comment by corinagum** [comment]
Conversation with @compulim: 

Do we have demand for changing the font other than the primary font? if we do, then we should consider implementing this. 

My initial assumption is that there hasn't been demand for this. I'm going to change the title of this issue to only refer to fonts and we can see if users add their +1, then go from there.

**Comment by corinagum** [comment]
All that's left for this one is initials color.

**Comment by compulim** [comment]
This can be done in the new "customizable per-message avatar" feature, released in 4.8.0.
